code/crates/test/framework/src/node.rs

```diff
@@ -48,6 +49,7 @@ where
     pub steps: Vec<Step<Ctx, State>>,
     pub state: State,
     pub middleware: Arc<dyn Middleware>,
+    pub config_customizer: Option<ConfigCustomizer<malachitebft_test_app::config::Config>>,
```

Comment:
We can maybe get rid of the `Option` here by setting this field to `Arc::new(|config| config)` by default.